test/fixtures/resolve.config.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
Is the removal of this file concious?

Reply:
> ... and removed unused file

yes, not used. I previously had a functional test but it was too flacky to push.